src/server/project.ts

```diff
@@ -362,7 +362,7 @@ namespace ts.server {
             return map(this.program.getSourceFiles(), sourceFile => {
                 const scriptInfo = this.projectService.getScriptInfoForPath(sourceFile.path);
                 if (!scriptInfo) {
-                    Debug.assert(false, `scriptInfo for a file '${sourceFile.fileName}' is missing.`);
+                    Debug.assert(false, "scriptInfo for a file is missing:", sourceFile.fileName);

```

Comment:
No need to change this, other than from an `assert(false, ` to `fail` as the condition was already checked.